lib/common-types.ts

```diff
@@ -154,6 +154,9 @@ export interface CollectionProp<TObj> {
   skip: number
   limit: number
   items: TObj[]
+  includes?: {
```

Comment:
This feels worth a dedicated type instead of adding it to the default one. And in case of the references endpoint is it really optional?

Reply:
What I understand is all `Collection` type apis have optional `includes` field, no?

Reply:
I don't think all of them have `includes` parameter